feature-libs/organization/administration/components/shared/sub-list/sub-list.component.html

```diff
@@ -21,7 +21,14 @@
         </cx-table>
       </section>
 
-      <div class="footer" *ngIf="data.pagination?.totalPages > 1">
+      <div
+        class="footer"
+        *ngIf="
+          data.pagination &&
+          data.pagination.totalPages !== undefined &&
```

Comment:
why `!== undefined` just for `totalPages`.  Could we just check if it's falsy?

Reply:
`totalPages` is `number`. If it is `0`, it is also `false`, but we also check `data.pagination.totalPages > 1`, so without `!== undefined` is also OK.

Reply:
Oh you're right.  Good catch.   Maybe there is something to be done with `??` if we need to cover `null` as well.  Otherwise leave it like this.